server/server.go

```diff
@@ -261,7 +386,7 @@ func (s *Server) handleStream(ctx context.Context, authCtx context.Context, conn
 		s.logger.Debug().
 			Str("target", addr.String()).
 			Str("source", source).
-			Msg("juicity received a udp request")
+			Msg("juicity received an [udp] request")
```

Comment:
```suggestion
			Msg("juicity received a [udp] request")
```

Actually, we should stick to a instead of an because udp is pronounced as ju:dipi.

Reply:
Nice catch. Solved. Thanks!